lib/index.ts

```diff
@@ -176,8 +182,8 @@ export default class Pageres<DestValue: string> extends EventEmitter {
 		}
 	}
 
-	save(streams: Array<PageresStream>) {
-		const files = [];
+	async save(streams: PageresStream[]) {
+		const files : any[] = [];
```

Comment:
Don't use `any`.

Reply:
Would it be worth enforcing this in the tsconfig?

Reply:
Yes, but hard to enable `noImplicitAny` if you don't control all the dependencies. But we could try, sure.